### PR TITLE
Sentry: Add AsyncioIntegration, make all integrations disabled

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cwsgiutils/sentry.py [hunk 1]
```diff
@@ -73,20 +75,43 @@ def includeme(config: Optional[pyramid.config.Configurator] = None) -> None:
...
-            ).upper(),
-        )
         traces_sample_rate = float(
             config_utils.env_or_config(
                 config, "SENTRY_TRACES_SAMPLE_RATE", "c2c.sentry_traces_sample_rate", "0.0"
             )
         )
+        integrations: list[sentry_sdk.integrations.Integration] = []
+        if config_utils.env_or_config(
+            config, "SENTRY_INTEGRATION_LOGGING", "c2c.sentry_integration_logging", "true"
```
Comment (gberaudo) [inline c2cwsgiutils/sentry.py:85]: What about having all these integrations disabled by default?

File: c2cwsgiutils/sentry.py [hunk 2]
```diff
@@ -73,20 +75,43 @@ def includeme(config: Optional[pyramid.config.Configurator] = None) -> None:
...
+        ).lower() in ("true", "1", "on", "yes"):
```
Comment (gberaudo) [inline c2cwsgiutils/sentry.py:86]: See `config_bool()`

**Review by gberaudo — APPROVED** [review]

File: c2cwsgiutils/sentry.py (hunk 1 above)
Comment (sbrunner) [reply c2cwsgiutils/sentry.py:85]: I don't know currently they allways be on, did we get issue by having them on?

File: c2cwsgiutils/sentry.py (hunk 2 above)
Comment (sbrunner) [reply c2cwsgiutils/sentry.py:86]: Done, thanks :-)

**Pull request merged**